db/migrate/20180802145655_add_address_to_schools.rb

```diff
@@ -0,0 +1,12 @@
+class AddAddressToSchools < ActiveRecord::Migration
+  def change
+    add_column :schools, :address_line_1, :string
+    add_column :schools, :address_line_2, :string
+    add_column :schools, :state, :string
+    add_column :schools, :postal_code, :string
+    add_column :schools, :phone_number, :string
+    add_column :schools, :borough, :string
+    add_column :schools, :created_at, :datetime
+    add_column :schools, :updated_at, :datetime
+  end
+end
```

Comment:
I'm a bit confused, because I also see this code in https://github.com/NYPL/MyLibraryNYCApp/pull/10/files.  I made some comments on it there.

Reply:
@daryanypl our team goal was to do smaller PRs, and PR 10 built on this one and this one wasn't approved & merged into development yet, so the code for this one had to be in both places.